### DISCUSSION TITLE
E-mails are going to spam

### DISCUSSION THREAD

**Discussion by kristoftorok** [plausible/analytics#1874] [Self-Hosted Support]
Hello. I use Mailcow as my email server. If I send an email from the SOGo webmail interface to my protonmail address, the email does not go to spam. And in the protonmail headers, the X-spam-score header says 0.

When plausible sends some email, whether I get a password reset or stats email it goes to spam and the X-spam-score header says 5.

In the config file, I put `SMTP_HOST_SSL_ENABLED=true` but it didn't get any better.

Does anyone have any idea why this is happening?

**Comment by neu-ma-tic** [comment]
It's likely your host ip range is flagged

**Reply by kristoftorok** [reply]
No. Because then all emails from all domains hosted from this mail server would be spam. But that's not the case. Only the emails that plausible sends out are spam.

**Comment by grobmeier** [comment]
Sending emails is supertricky. You have to set DKIM, SPF etc headers on your domains DNS to have this done well. Also, the senders email address needs to match up - some stuff to take care of.

Looking at Mailcow there might be some stuff which might go wrong with the selfhosted version. I would recommend using a hosted version where your domain is having all the right settings, or to use well working service like AWS SNS or (what I use): Postmarkapp

If you don't want to use those services, you should look into the headers of the original email you received and check what is inside.

Cheers,
Christian

**Reply by kristoftorok** [reply]
Thanks for the answer!

Well, running a mail server is not easy. However, this is not the first time I have run a mail server. I have DMARC,DKMIN,SPF and even rDNS configured. 

With all emails not sent out by plausible there is no problem.

I did a little digging into how plausible works and it is probably connecting to the mail server unencrypted. This is probably causing the problem.

**Reply by grobmeier** [reply]
OK, if you got all that stuff done it should be fine. With that background I assume you already checked if you server is somehow mentioned on a black list? https://dnschecker.org/ip-blacklist-checker.php
Of course you said other emails went through fine, but maybe better double check in this case.
Can you also see something on the raw message headers? If possible, maybe post them here too

**Reply by kristoftorok** [reply]
Thanks for the reply. In a post below we have probably found the problem. It seems plausible is not formatting emails correctly.

**Comment by tpimh** [comment]
One tool I found very useful is mail-tester.com. Could you please send the offensive message to that service (you can change your email address in the database, then requect the password reset, then change the address back) and post your results?

**Reply by kristoftorok** [reply]
The email that plausible sent is 7.2
The one I sent as a test is 10

![Screenshot from 2022-05-18 17-42-58](https://user-images.githubusercontent.com/55440418/169085189-5a7958b3-8998-439c-ad4a-dd5e5770403e.png)
![Screenshot from 2022-05-18 17-42-53](https://user-images.githubusercontent.com/55440418/169085194-8ce641e7-6732-4218-8844-a97905509671.png)
![plausibel](https://user-images.githubusercontent.com/55440418/169085196-3276d78e-5de4-407d-bb9f-e598644c570e.png)
